compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIETileAndFuse.cpp

```diff
@@ -108,6 +111,9 @@ void AMDAIETileAndFusePass::runOnOperation() {
     // the skip ops list which currently contains linalg.copy and tensor.unpack.
     if (op.getLoopIteratorTypes().empty() || consumerToSkip(op))
       return WalkResult::advance();
+    if (!tileElementwise && isa<linalg::GenericOp>(op) &&
```

Comment:
Can you add some comments as to what this is doing?

Reply:
Refactored this part of codes and added some comments.